doc/articles/uno-development/creating-ui-tests.md

```diff
@@ -17,11 +17,12 @@ UI tests contribute significantly to the CI build time, and for many purposes a
 ## Running UI tests locally
 
 1. Ensure [your environment is configured](../get-started-vs.md) for the platform you want to run on.
-2. Open Uno.UI with the [correct target override and solution filter](building-uno-ui.md) for the platform you want to run on.
-3. [Build and run the SamplesApp](working-with-the-samples-apps.md) at least once. 
-4. Open the [`Constants.cs`](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/Constants.cs) file and change the `CurrentPlatform` field to the platform you want to run on. Only Android and WASM are supported from Visual Studio for Windows. (Running tests on iOS using a Mac is possible, see additional instructions below.)
-5. Open the [Test Explorer](https://docs.microsoft.com/en-us/visualstudio/test/run-unit-tests-with-test-explorer) in Visual Studio.
-6. UI tests are grouped under 'SamplesApp.UITests'. From the Test Explorer you can run all tests, debug a single test, etc. 
+1. Open Uno.UI with the [correct target override and solution filter](building-uno-ui.md) for the platform you want to run on.
+1. [Build and run the SamplesApp](working-with-the-samples-apps.md) at least once. 
+1. Open the [`Constants.cs`](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/Constants.cs) file and change the `CurrentPlatform` field to the platform you want to run on. Only Android and WASM are supported from Visual Studio for Windows. (Running tests on iOS using a Mac is possible, see additional instructions below.)
+1. If testing on WebAssembly, ensure that [`WebAssemblyDefaultUri `](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/Constants.cs) matches Url used when the sample app was launched in the step above. Visual Studio may change the Url on demand to avoid conflicts with already running sites on the same machine.
```

Comment:
nit: extra space

```suggestion
1. If testing on WebAssembly, ensure that [`WebAssemblyDefaultUri`](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/Constants.cs) matches Url used when the sample app was launched in the step above. Visual Studio may change the Url on demand to avoid conflicts with already running sites on the same machine.
```